test/unit/data/datatypes/test_castep.py

```diff
@@ -0,0 +1,26 @@
+from galaxy.datatypes.text import Castep, FormattedDensity, Param
+from .util import get_input_files
+
+
+def test_castep():
```

Comment:
These tests can be moved to doctests, e.g. https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/datatypes/text.py#L61